doc/userguide/src/CreatingTestData/CreatingUserKeywords.rst

```diff
@@ -702,9 +702,9 @@ succeeds:
 
 Before looking which match is best, Robot Framework checks are some of the matching
 keywords implemented in the same file as the caller keyword. If there are such keywords,
-they are given precedence over other keywords. If there are still conflicts
-after looking for best matches, Robot Framework checks can they be
-resolved based on the `library search order`_.
+they are given precedence over other keywords. Alternatively, `library search order`_
```

Comment:
Link does not seem to work. Could be replaced with `:name:`Set Library Search Order` from the BuiltIn_ library`